Gems/PhysX/Code/Editor/Source/ComponentModes/Joints/JointsComponentMode.cpp

```diff
@@ -90,7 +90,7 @@ namespace PhysX
 
         void RefreshUI()
         {
-            // The reason this is in a free function is because ColliderComponentMode
+            // The reason this is in a free function is because JointsComponentMode
             // privately inherits from ToolsApplicationNotificationBus. Trying to invoke
             // the bus inside the class scope causes the compiler to complain it's not accessible
             // to due private inheritance.
```

Comment:
nitpick - know this isn't your change but there's a typo here. Think it should be "due to" rather than "to due"

Reply:
nice! 
fixed